app/src/main/java/com/sopt/now/presentation/auth/signup/SignUpActivity.kt

```diff
@@ -0,0 +1,82 @@
+package com.sopt.now.presentation.auth.signup
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import com.google.android.material.snackbar.Snackbar
+import com.sopt.now.R
+import com.sopt.now.data.User.UserDao
+import com.sopt.now.data.User.UserData
+import com.sopt.now.databinding.ActivitySignupBinding
+import com.sopt.now.presentation.auth.login.LoginActivity
+
+class SignUpActivity : AppCompatActivity()  {
+    private val binding by lazy { ActivitySignupBinding.inflate(layoutInflater) }
+    private val viewModel: SignUpViewModel by viewModels()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+
+        clicksignUpButton()
+    }
+
+    private fun clicksignUpButton() {
+        binding.btSignupButton.setOnClickListener {
+            val user = getUserInfo()
+            // 유효성 검사 후 회원가입 진행
+            setSignupResult(user)
+        }
+    }
+
+    private fun setSignupResult(user: UserData) {
+        val userDao : UserDao? = null
+        val validateSignUpData = viewModel.validateSignUp(user)
+        if(validateSignUpData == null){
```

Comment:
```suggestion
        if(viewModel.validateSignUp(user)
 == null){
```
변수를 여기서만 사용한다면 바로 넣어줘도 될 것 같아요!